tests/scenarios/PowershellScriptTest/config.json

```diff
@@ -9,31 +9,36 @@
             "executable": "PowershellScriptTest.exe",
 			"startScript":
 			{
-				"scriptPath": "SayHello.ps1"
+				"scriptPath": "SayHello.ps1",
+				"wait": true

```

Comment:
>wait [](start = 5, length = 4)

I guess the earlier comment in the readme was to make this configuration parameter a little more explicit. "RunScriptAsync" or something like that instead of "Wait"